services/integration/src/routes/constitution.routes/constitution.js

```diff
@@ -1,5 +1,5 @@
 const Router = require('express').Router();
-const constitution = require('../controllers/constitution');
+const constitution = require('../../controllers/constitution.controllers/constitution');
```

Comment:
Same error with this file name, folder names should not be like this `constitution.controllers`